### ISSUE TITLE
Шаг #1

### ISSUE THREAD

**Issue by odesskij** [polovnik/phone-cat#1]
У тебя связанность между html и js. В шаблоне у тебя стоит ul, а js делает li.
Если у тебя сам компонент задуман как список, лучше вынести ul, как его часть. 
https://github.com/polovnik/phone-cat/blob/master/js/main.js#L36 у тебя возникает побочный эффект, ты завязываешься на элемент(#list), который не можешь контролировать.
Также из-за того, что твое приложение не контролирует контейнер, есть два раза вызвать `new Phone(phones).render();`, у тебя будет в два раза больше плиток. 

Сделай как в backbone, у тебя есть какое-то View, ты в него передаешь el (контейнер), в который ты уже рендеришь свой компонент. 
Типа `const view = new PhoneView(el, phones); view.render();`

Стилистика: 
https://github.com/polovnik/phone-cat/blob/master/js/main.js#L67 не забывай про стрелочные функции. 
Комментарии в начале файла, обычно мусор не коммитят. Если он тебе больше не нужен, посто удаляй его.

**Comment by tomfun** [comment]
Согласен. Компонент, который рисует телефон, не должен знать о том, где он будет находиться. Такая вот идеология. Чем меньше компоненты связаны, тем лучше.

> Сделай как в backbone, у тебя есть какое-то View, ты в него передаешь el (контейнер), в который ты уже рендеришь свой компонент.
> Типа const view = new PhoneView(el, phones); view.render();
 - или в бекбоне вью сама создаёт элемент, с которым работает, но этот элемент нигде не появляется, пока явно не скажешь куда запихнуть.
